supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalRuntimeAttributesSpec.scala

```diff
@@ -107,6 +110,28 @@ class LocalRuntimeAttributesSpec extends WordSpecLike with Matchers {
       assertLocalRuntimeAttributesSuccessfulCreation(runtimeAttributes, workflowOptions, staticDefaults.copy(continueOnReturnCode = ContinueOnReturnCodeSet(Set(1, 2))))
     }
 
+    "use reasonable default values" in {
+      val expectedRuntimeAttributes = staticDefaults
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { }""").head
+      assertLocalRuntimeAttributesSuccessfulCreation(runtimeAttributes, emptyWorkflowOptions, expectedRuntimeAttributes)
+    }
+
+    "warn for unrecognized keys" in {
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { whatIsThis: "noIdea" andThis: "donno" }""").head
+      val mockLogger = mock[Logger]
+      mockLogger.warn(anyString).answers { _ match {
+          case s: String =>
```

Comment:
And to take it a step further, if the rhs of a case is too complicated that one has to worry about single letter placeholders, the rhs is too complicated and should be pulled into a function